src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -188,6 +187,74 @@
   gap: $margin--small;
 }
 
+.participants-reset-state-banner__container {
+  margin-top: auto; // position at the very bottom
+  margin-left: -$settings-dialog-container--sides-small;
+  padding: $padding--small $padding--medium; // create some space
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+
+  width: calc(100% + 2 * $settings-dialog-container--sides-small); // overflow over parent margin
+
+  background-color: $color-lighter-gray;
+
+  transform: translateY(100%);
+  transition: transform ease-in-out 0.3s;
+}
+
+.participants-reset-state-banner__container--is-active {
+  transform: translateY(0);
+}
+
+.participants-reset-state-banner__icon-and-text {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  column-gap: 4px;
+}
+
+.participants-reset-state-banner__check-icon {
+  width: $icon--large;
+  height: $icon--large;
+  min-width: $icon--large;
+  min-height: $icon--large;
+}
+
+.participants-reset-state-banner__text {
+  color: $menu-icon-background-color--dark;
+  font-weight: 600;
+}
+
+.participants-reset-state-banner__button {
+  padding: 0 $padding--medium;
+
+  height: 36px;
+  background-color: $menu-icon-background-color--dark;
+  color: $color-white;
+  border: none;
+  border-radius: $rounded--full;
+  font-weight: bold;
+  font-size: 14px;
```

Comment:
we also have `$text-size--small` and `$text-size--medium` if you want to use them.

Reply:
I ended up removing the line setting the font-size because it doesn't really seem to matter